libc/src/stdio/printf_core/CMakeLists.txt

```diff
@@ -1,3 +1,25 @@
+if(LIBC_CONF_PRINTF_DISABLE_FLOAT)
```

Comment:
if these conditionals are being moved into `printf_core` should the `add_subdirectory` lines in stdio be moved back up to the top?

Reply:
Where do `add_subdirectory` in `stdio` becomes irrelevant from the config options point of view - it should be at a place where it makes the most practical sense.